UnitsNet/GeneratedCode/Quantities/Acceleration.g.cs

```diff
@@ -736,9 +736,10 @@ public static string GetAbbreviation(AccelerationUnit unit)
 
         public int CompareTo(object obj)
         {
-            if (obj == null) throw new ArgumentNullException("obj");
-            if (!(obj is Acceleration)) throw new ArgumentException("Expected type Acceleration.", "obj");
-            return CompareTo((Acceleration) obj);
+            if(obj is null) throw new ArgumentNullException("obj");
+            if(!(obj is Acceleration)) throw new ArgumentException("Expected type Acceleration.", "obj");
```

Comment:
Let's add `nameof(obj)` too.

Reply:
Didn't know you could do this either 👍 

Done!